### ISSUE TITLE
more images for object detection, worse results ?!

### ISSUE THREAD

**Issue by JVR32** [NVIDIA/DIGITS#1361]
Hello,

I trained a network for the detection of an object class that I called '110'.
I started with 1862 images (training + validation), and I ended up with 7382 images (training + validation).
My expectation was that the recall and precision should improve when more images were used.
During the first experiments, that was indeed the case. But at a certain point, augmenting the number of images (used for training + validation) didn't improve the results anymore, it's the opposite that's true.

![precision_110](https://cloud.githubusercontent.com/assets/19925967/21466036/1c3a06d2-c9ba-11e6-81f0-895cdef74af1.png)

In that last experiment with 7382 images, I used  5906 images for training and 1476 images for validation. Then I suspected the number of validation images to be to low compared to the number of training images. So I added some extra validation images, and I repeated the experiment with the same training images (5906), but now with 2391 validation images.  But instead of improving the results, again it gets worse ...

![precision_110](https://cloud.githubusercontent.com/assets/19925967/21466076/b3c134ac-c9bb-11e6-8487-fac366946ea9.png)

Does anyone have an idea why adding extra images degrades the detection instead of improving it?
And what can I try to get better detection?

For the training, I used the following settings :

![train_settings](https://cloud.githubusercontent.com/assets/19925967/21466089/83e8b0ba-c9bc-11e6-8ae6-4653c645ffe1.JPG)

Regards,

J.

**Comment by pkrush** [comment]
Can this mean overfitting is going down as you increase the amount of images you’re training on?

I have noticed this trend with simple classification. Without seeing a third set of images on the graph, the test set, it’s hard to tell if this is the case.

**Comment by JVR32** [comment]
No. I first noticed this trend when augmenting the number of training images from 6075 to 7382. I assumed detection on test images (not used for training or validation) would improve, but it got worse. That's why I now tried to augment the number of validation images, because I suspected the number of validation images to be too small compared to the number of training images. But, no improvement when doing that.

So ... when I start with a small number of training/validation images, detection is not that good (logical of course). Augmenting the number of images used for training/validation improves the detection results. But at a certain point, augmenting the number of images gives worse results than before. So I guess I won't be able to get better results by simply adding more images, but that I also will have to change certain parameters. But I have no idea which parameters ...

**Comment by alpop** [comment]
It does not look abnormal to me. Even with  smaller training data one can achieve somewhat better performance just by luck coming from combination of  specific  training/validation split and learning parameters. For instance,  If you did  k-fold cross-validation and/or used different learning options, you might have observed different behaviors/trends. And surely there may be errors in training labels and algorithms, too.
